bin/node/cli/Cargo.toml

```diff
@@ -155,8 +157,13 @@ cli = [
 	"sc-finality-grandpa-warp-sync",
 	"structopt",
 	"substrate-build-script-utils",
+	"try-runtime-cli",
 ]
 runtime-benchmarks = [
 	"node-runtime/runtime-benchmarks",
 	"frame-benchmarking-cli",
 ]
+try-runtime = [
```

Comment:
+1 more descriptive name is needed imo

Reply:
I convinced shawn offline to keep it as-is for now because I think I will add other use cases in the realm of testing runtimes, so something like `debug-runtime-migration` or so would also be too restrictive. 

Shall we rename later, if needed, @bkchr?

Reply:
As I said, adding at least a comment would be nice,